feature-libs/checkout/components/guards/checkout-auth.guard.ts

```diff
@@ -9,9 +9,9 @@ import {
   GlobalMessageService,
   GlobalMessageType,
   SemanticPathService,
-  User,
-  UserService,
+  User as CoreUser,
```

Comment:
Can we use only model from @spartacus/core for now? Do we need 2 different interfaces?

Reply:
yep, switched to the user from the user lib